modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -210,6 +210,34 @@ protected void doServeResource(
 		).build();
 	}
 
+	private Collector<AssetCategory, ?, Map<Long, Map<String, Object>>>
+		_collectVocabularies(Locale locale) {
```

Comment:
I think that if we follow Liferay name guides, this should be called _getCollector(